views/package_format_json.dt

```diff
@@ -583,6 +583,10 @@ block body
 			td "betterC"
 			td Compile in betterC mode
 			td -betterC
+		tr
+			td "lowmem"
+			td Enable the garbage collector for the compiler(dmd/ldc), reducing the compiler memory requirements but increasing compile times.
```

Comment:
cc @ibuclaw does GDC support `-lowmem`?

Reply:
I don't export it as something controllable by the command-line, so `-lowmem` is the default.

Reply:
At some point in the future I hope to be able to do one better, and call `rt_term()` after the codegen pass, so all front-end memory is freed before passing to the back-end, there's still a little bit of interdependencies between gcc trees and dmd ast though.